<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExternalInput</class>
 <widget class="QMainWindow" name="ExternalInput">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>867</width>
    <height>634</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPlainTextEdit" name="commandEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>781</width>
      <height>61</height>
     </rect>
    </property>
    <property name="plainText">
     <string/>
    </property>
   </widget>
   <widget class="QSpinBox" name="inputBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>61</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="enableBox">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>98</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>enable</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="outputBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>781</width>
      <height>331</height>
     </rect>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>781</width>
      <height>27</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Choose preset...</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>avconv -f video4linux2 -i /dev/video0 -s %1x%2 -pix_fmt rgb32 -f rawvideo -</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>avconv -i http://10.0.0.2:9981/stream/channelid/74265269 -s %1x%2 -pix_fmt rgb32 -f rawvideo -</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>avconv -i rtsp://10.0.0.152:8086 -max_delay 0 -s %1x%2 -pix_fmt rgb32 -f rawvideo -</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>avconv -f mjpeg -i http://10.0.0.152:8085 -max_delay 0 -s %1x%2 -pix_fmt rgb32 -f rawvideo -</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>867</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
